### PR TITLE
Move bibliography to rustc-guide

### PR DESCRIPTION
**Pull request by mark-i-m**
https://github.com/rust-lang/rustc-guide/pull/349

cc @spastorino

r? @pietroalbini

### CODE REVIEW

**Comment by spastorino** [comment]
@mark-i-m I'm in favor of adding the bibliography to rustc-guide but unsure about removing it from here.
@nikomatsakis ?

**Comment by pietroalbini** [comment]
Can you add a redirect?

**Comment by spastorino** [comment]
Yeah, or the link to the bibliography on forge could just be a link to be bibliography on rustc-guide.

**Comment by pietroalbini** [comment]
Sure, but I'd also prefer the redirect to be in place anyway, to prevent broken links.

File: index.md [hunk 1]
```diff
@@ -50,7 +50,6 @@ PRs against [rust-lang/rust-forge].
 ### Other things
 
 * [Friend of the Tree archives](fott.html).
-* [Bibliography](bibliography.html). Research papers and other links to projects that influenced Rust. Papers about Rust.
```
Comment (spastorino) [inline index.md]: So, let's link this to rustc-guide bibliography and add a redirect from the forge bibliography to the rustc-guide one.

**Comment by mark-i-m** [comment]
Done. I need to fix some broken links in the other PR before we merge this though...

**Comment by mark-i-m** [comment]
Ok, the other PR has merged. I think this is ready.

**Pull request merged**